contrib/collectd/patches/fix_clang_warning.patch

```diff
@@ -0,0 +1,94 @@
+--- a/src/bind.c	2024-07-22 14:42:25.365983474 +0000
```

Comment:
you don't need to patch warnings

Reply:
they treat warnings as errors, this is a patch they have for 6.0 but haven't backported to 5.12.0

Reply:
ah, in that case you should remove what sets Werror instead

Reply:
done